src/ripple/overlay/impl/OverlayImpl.cpp

```diff
@@ -714,21 +662,23 @@ OverlayImpl::onManifests (
     {
         auto& s = m->list ().Get (i).stobject ();
 
-        if (auto mo = make_Manifest (s))
+        if (auto mo = Manifest::make_Manifest (s))
         {
             uint256 const hash = mo->hash ();
             if (!hashRouter.addSuppressionPeer (hash, from->id ()))
                 continue;
 
             auto const serialized = mo->serialized;
-            auto const result = manifestCache_.applyManifest (
+            auto const result = app_.manifestCache ().applyManifest (
                 std::move(*mo),
-                app_.validators(),
-                journal);
+                app_.validators());
 
             if (result == ManifestDisposition::accepted ||
                     result == ManifestDisposition::untrusted)
-                app_.getOPs().pubManifest (*make_Manifest(serialized));
+            {
+                app_.getOPs().pubManifest (
+                    *Manifest::make_Manifest(serialized));
```

Comment:
`error: use of deleted function 'ripple::Manifest::Manifest(const ripple::Manifest&)'`
It looks like we would need to replace the `std::move(*mo)` with 
`Manifest (mo->serialized, mo->masterKey, mo->signingKey, mo->sequence)`
or finally add a copy constructor to `Manifest`.
Or we can leave this as is.